.changeset/config.json

```diff
@@ -11,6 +11,7 @@
   "updateInternalDependencies": "patch",
   "ignore": [],
   "___experimentalUnsafeOptions_WILL_CHANGE_IN_PATCH": {
```

Comment:
😯 Ahah looked for docs on `useCalculatedVersionForSnapshots` and I guess this explains why my Google search came up empty! 👍

Reply:
😄 Info here: https://github.com/hashicorp/nextjs-scripts/pull/204#discussion_r697616836

Reply:
oh nice, missed that earlier 🙇 ty!